ejerciciosPilas/src/comportamientos/ComportamientoConVelocidad.ts

```diff
@@ -11,33 +11,37 @@
  *               Como esto juega con la animacion, es preferible no tocarlo.
  */
 class ComportamientoConVelocidad extends ComportamientoAnimado {
+
+	static modoTurbo = false
+
 	_pasosRestantes;
 	vueltasSinEjecutar;
 	enQueVueltaEjecuto;
 
-	preAnimacion(){
+	preAnimacion() {
 		super.preAnimacion();
 		this.argumentos.cantPasos = this.argumentos.cantPasos || 10;
-		this.argumentos.velocidad = this.argumentos.velocidad || 20;
+		this.argumentos.velocidad = (ComportamientoConVelocidad.modoTurbo) ? 100 : this.argumentos.velocidad || 20;
+		this.argumentos.deboCortarAnimaciones = (ComportamientoConVelocidad.modoTurbo) ? true : this.argumentos.deboCortarAnimaciones || false;
```

Comment:
Idem ant. Además creo que se puede escribir: `(ComportamientoConVelocidad.modoTurbo) || this.argumentos.deboCortarAnimaciones || false;`

Reply:
Idem anterior, pero acá además me parece que tenés razón respecto de la expresión, cambiémoslo.

Reply:
Me gusta la idea